### ISSUE TITLE
Unexpected keyword argument 'proxy_forwarded_proto_header' with daphne==1.4.0

### ISSUE THREAD

**Issue by matthiaskoenig** [django/channels#816]
The latest release of daphne breaks channels due to some changes in keywords. On ubuntu linux, everything worked until the daphne release an hour ago.
```
python manage.py runserver
```
reslts in
```
Performing system checks...

System check identified no issues (0 silenced).
January 02, 2018 - 20:47:50
Django version 2.0.1, using settings 'teweb.settings'
Starting Channels development server at http://127.0.0.1:8000/
Channel layer default (asgi_redis.core.RedisChannelLayer)
Quit the server with CONTROL-C.
2018-01-02 20:47:50,654 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
2018-01-02 20:47:50,654 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
2018-01-02 20:47:50,655 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x7fce8096a598>
Traceback (most recent call last):
  File "/home/mkoenig/envs/tellurium-web/lib/python3.5/site-packages/django/utils/autoreload.py", line 225, in wrapper
2018-01-02 20:47:50,656 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
    fn(*args, **kwargs)
  File "/home/mkoenig/envs/tellurium-web/lib/python3.5/site-packages/channels/management/commands/runserver.py", line 100, in inner_run
    websocket_handshake_timeout=self.websocket_handshake_timeout,
  File "/home/mkoenig/envs/tellurium-web/lib/python3.5/site-packages/daphne/server.py", line 101, in run
    websocket_handshake_timeout=self.websocket_handshake_timeout
TypeError: __init__() got an unexpected keyword argument 'proxy_forwarded_proto_header'
```

Pip requirements are:
```
amqp (2.2.2)
antimony (2.9.4)
appdirs (1.4.3)
asgi-redis (1.4.3)
asgiref (1.1.2)
attrs (17.4.0)
autobahn (17.10.1)
Automat (0.6.0)
billiard (3.5.0.3)
celery (4.1.0)
certifi (2017.11.5)
channels (1.1.8)
chardet (3.0.4)
constantly (15.1.0)
coreapi (2.3.3)
coreschema (0.0.4)
cycler (0.10.0)
daphne (1.4.0)
decorator (4.1.2)
Django (2.0.1)
django-celery-beat (1.1.0)
django-celery-results (1.0.1)
django-choices (1.6.0)
django-debug-toolbar (1.9.1)
django-filter (1.1.0)
django-model-changes-py3 (0.14.1)
django-rest-swagger (2.1.2)
djangorestframework (3.7.7)
ephem (3.7.6.0)
gunicorn (19.7.1)
hyperlink (17.3.1)
idna (2.6)
incremental (17.5.0)
ipykernel (4.7.0)
ipython (6.2.1)
ipython-genutils (0.2.0)
isodate (0.6.0)
itypes (1.1.0)
jedi (0.11.1)
Jinja2 (2.10)
jsonschema (2.6.0)
jupyter-client (5.2.0)
jupyter-core (4.4.0)
kombu (4.1.0)
libroadrunner (1.4.24)
Markdown (2.6.10)
MarkupSafe (1.0)
matplotlib (2.1.1)
msgpack-python (0.4.8)
nbformat (4.4.0)
numpy (1.13.3)
openapi-codec (1.3.2)
pandas (0.22.0)
parso (0.1.1)
pexpect (4.3.1)
phrasedml (1.0.9)
pickleshare (0.7.4)
pip (9.0.1)
plotly (2.2.3)
pluggy (0.6.0)
prompt-toolkit (1.0.15)
psycopg2 (2.7.3.2)
ptyprocess (0.5.2)
py (1.5.2)
Pygments (2.2.0)
pyparsing (2.2.0)
pytest (3.3.1)
python-dateutil (2.6.1)
python-magic (0.4.15)
pytz (2017.3)
pyzmq (16.0.3)
rdflib (4.2.2)
redis (2.10.6)
requests (2.18.4)
rrplugins (1.1.10)
sbml2matlab (0.9.1)
scipy (1.0.0)
setuptools (38.2.5)
simplegeneric (0.8.1)
simplejson (3.13.2)
six (1.11.0)
sqlparse (0.2.4)
tecombine (0.2.2.1)
tellurium (2.0.13)
tenuml (1.1.1)
tesbml (5.15.0.1)
tesedml (0.4.3)
tornado (4.5.2)
traitlets (4.3.2)
Twisted (17.9.0)
txaio (2.8.2)
uritemplate (3.0.0)
urllib3 (1.22)
vine (1.1.4)
wcwidth (0.1.7)
wheel (0.30.0)
zope.interface (4.4.3)
```

Workaround is to downgrade daphne by using
```
daphne==1.3.0
```
in requirements.txt.

Best M

**Comment by andrewgodwin** [comment]
Fixed in release 1.4.1.

**Comment by matthiaskoenig** [comment]
Thanks a lot for the quick fix. Works.
